src/rql/index.ts

```diff
@@ -109,17 +120,59 @@ export function rqlBuilder(rql?: RQLString | string): RQLBuilder {
     },
   };
 
-  function processQuery(operation: string, value: string) {
-    if (value.includes(' ')) {
-      console.log(
-        'A space has been detected while building the rql, please be aware that problems can arise'
-      );
+  // Resolve the input for a rql builder
+  function determineInput() {
```

Comment:
(opinion, no need to change!) If we accept `input` as a parameter, this function could be removed from the inner scope of the "rqlBuilderFactory". In my opinion making it easier to process what is going on within this function and more clear in its purpose when it is used at line 13.

Reply:
Absolutely I had initially done it this way, but I thought it perhaps was nicer to have all the variables it was using being scoped within it's own function. If that's your preference, Im actually happy to extract it back out.

Reply:
I'm a fan 👌 Nice "pure" function, easy to follow and reason about